pkg/services/ngalert/store/alert_rule.go

```diff
@@ -64,20 +64,30 @@ func getAlertRuleByUID(sess *sqlstore.DBSession, alertRuleUID string, orgID int6
 }
 
 // DeleteAlertRuleByUID is a handler for deleting an alert rule.
-func (st DBstore) DeleteAlertRuleByUID(ctx context.Context, orgID int64, ruleUID string) error {
+func (st DBstore) DeleteAlertRulesByUID(ctx context.Context, orgID int64, ruleUID ...string) error {
 	return st.SQLStore.WithTransactionalDbSession(ctx, func(sess *sqlstore.DBSession) error {
-		_, err := sess.Exec("DELETE FROM alert_rule WHERE org_id = ? AND uid = ?", orgID, ruleUID)
+		params := "(" + strings.Repeat(",?", len(ruleUID))[1:] + ")"
+
+		args := make([]interface{}, 0, len(ruleUID)+2)
+		args = append(args, "DELETE FROM alert_rule WHERE org_id = ? AND uid IN "+params)
+		args = append(args, orgID)
+		for _, arg := range ruleUID {
+			args = append(args, arg)
+		}
+
+		_, err := sess.Exec(args...)
```

Comment:
Also not for this pr, but I'm surprised we don't differentiate the query string itself from the query arguments ...